decidim-multisurveys/lib/decidim/multisurveys/component.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "decidim/components/namer"
+
+Decidim.register_component(:multisurveys) do |component|
```

Comment:
Here we have "la chicha": The component manifest, which contains the recipe to create components of this type.